src/modules/libraryModule.js

```diff
@@ -14,80 +14,127 @@ export default class Library {
 
   removeButton;
 
+  inputField;
+
+  checkboxButton;
+
   static pushToStorage = (obj) => {
     const stringify = JSON.stringify(obj);
-    sessionStorage.setItem('strShelf', stringify);
+    localStorage.setItem('strShelf', stringify);
   }
 
   static displayBook(title, id) {
     return `
-        <div class="box unchecked"></div>
-        <div class="task title-author">${title}</div>
-        <div class="handle removeButton" id="${id}" type="button"></div>
+    <div class="box unchecked"></div>
+    <input class="task" type="text" value="${title}" name="Item1">
+    <div class="handle removeButton" id="${id}" type="button"></div>
```

Comment:
- [ ] It would be better if our PR is edited to provide more detailed information. Kindly look at the project description and there are a few things you can add to the PR to make it more professional

Reply:
Done, thank you ;-)